Packs/CofenseTriage/Integrations/CofenseTriagev2/CofenseTriagev2.py

```diff
@@ -542,11 +542,13 @@ def search_inbox_reports_command(triage_instance) -> None:
         return
 
     try:
-        max_pages = int(demisto.getArg("max_matches") or 10) / 10  # TODO Triage's number of items per page
+        max_matches = int(demisto.getArg("max_matches")) or 10
     except ValueError:
         return_error("max_matches must be an integer if specified")
         return
 
+    max_pages = max_matches / 10  # Triage's number of items per page
```

Comment:
what happens if `max_matches / 10` isn't an integer?

Reply:
I believe Triage will handle it correctly, but just to be safe I changed the expression to `math.ceil(max_matches / 10)`.